agent/04_agent_configure.sh

```diff
@@ -110,6 +110,20 @@ cat >> "${MANIFESTS_PATH}/agent-cluster-install.yaml" << EOF
       hostPrefix: ${CLUSTER_HOST_PREFIX}
     serviceNetwork:
     - ${SERVICE_NETWORK}
+EOF
+case $IP_STACK in
+  v4v6 | v6)
+cat >> "${MANIFESTS_PATH}/agent-cluster-install.yaml" << EOF
+    networkType: OVNKubernetes
+EOF
+    ;;
+  *)
+cat >> "${MANIFESTS_PATH}/agent-cluster-install.yaml" << EOF
+    networkType: OpenShiftSDN
+EOF
+    ;;
+esac
+cat >> "${MANIFESTS_PATH}/agent-cluster-install.yaml" << EOF
```

Comment:
@rwsu what about setting the field just as in https://github.com/openshift-metal3/dev-scripts/blob/522d3c853a36c44125ea165cf8e633c5e88d7693/ocp_install_env.sh#L218? `NETWORK_TYPE` should already contain the right value based on IP_STACK

Reply:
Ah. Even better. I'll make that change. Thanks!